src/java.base/share/classes/java/util/Locale.java

```diff
@@ -70,131 +70,151 @@
 import sun.util.locale.provider.TimeZoneNameUtility;
 
 /**
- * A {@code Locale} object represents a specific geographical, political,
- * or cultural region. An operation that requires a {@code Locale} to perform
- * its task is called <em>locale-sensitive</em> and uses the {@code Locale}
- * to tailor information for the user. For example, displaying a number
- * is a locale-sensitive operation&mdash; the number should be formatted
- * according to the customs and conventions of the user's native country,
- * region, or culture.
- *
- * <p> The {@code Locale} class implements IETF BCP 47 which is composed of
+ * A {@code Locale} represents a specific geographical, political,
+ * or cultural region. An API that requires a {@code Locale} to perform
+ * its task is <em>locale-sensitive</em> and uses the {@code Locale}
+ * to tailor information for the user. These <em>locale-sensitive</em> APIs
```

Comment:
Either as part of this change or in follow-up work, please consider defining a user-index term for "locale
-sensitive".

Reply:
Thanks. (No update to the CSR is needed for adding the index term.)